modules/mobile/spec/request/letters_request_spec.rb

```diff
@@ -3,139 +3,92 @@
           'letters' =>
-                  [
-                    {
-                      'name' => 'Commissary Letter',
-                      'letterType' => 'commissary'
-                    },
-                    {
-                      'name' => 'Proof of Service Letter',
-                      'letterType' => 'proof_of_service'
-                    },
-                    {
-                      'name' => 'Proof of Creditable Prescription Drug Coverage Letter',
-                      'letterType' => 'medicare_partd'
-                    },
-                    {
-                      'name' => 'Proof of Minimum Essential Coverage Letter',
-                      'letterType' => 'minimum_essential_coverage'
-                    },
-                    {
-                      'name' => 'Service Verification Letter',
-                      'letterType' => 'service_verification'
-                    },
-                    {
-                      'name' => 'Civil Service Preference Letter',
-                      'letterType' => 'civil_service'
-                    },
-                    {
-                      'name' => 'Benefit Summary and Service Verification Letter',
-                      'letterType' => 'benefit_summary'
-                    },
-                    {
-                      'name' => 'Benefit Verification Letter',
-                      'letterType' => 'benefit_verification'
-                    }
-                  ]
+            [
+              {
+                'name' => 'Commissary Letter',
+                'letterType' => 'commissary'
+              },
+              {
+                'name' => 'Proof of Service Letter',
+                'letterType' => 'proof_of_service'
+              },
+              {
+                'name' => 'Proof of Creditable Prescription Drug Coverage Letter',
+                'letterType' => 'medicare_partd'
+              },
+              {
+                'name' => 'Proof of Minimum Essential Coverage Letter',
+                'letterType' => 'minimum_essential_coverage'
+              },
+              {
+                'name' => 'Service Verification Letter',
+                'letterType' => 'service_verification'
+              },
+              {
+                'name' => 'Civil Service Preference Letter',
+                'letterType' => 'civil_service'
+              },
+              {
+                'name' => 'Benefit Summary and Service Verification Letter',
+                'letterType' => 'benefit_summary'
+              },
+              {
+                'name' => 'Benefit Verification Letter',
+                'letterType' => 'benefit_verification'
+              }
+            ]
         }
       }
     }
   end
 
+  before do
+    token = 'abcdefghijklmnop'
+    allow_any_instance_of(Lighthouse::LettersGenerator::Configuration).to receive(:get_access_token).and_return(token)
+    allow(File).to receive(:read).and_return(rsa_key.to_s)
```

Comment:
We can actually delete this. I think you added it while trying to mock authentication.